lib/hammer_cli_foreman_remote_execution.rb

```diff
@@ -1,4 +1,10 @@
+require 'hammer_cli'
 require 'hammer_cli_foreman'
+require 'hammer_cli_foreman_tasks'
 
 module HammerCLIForemanRemoteExecution
+  require 'hammer_cli_foreman_remote_execution/options/normalizers'
```

Comment:
I've noticed the hammer was not displaying error messages from the validations. Adding this helped the situation.

```
  def self.exception_handler_class
    HammerCLIForeman::ExceptionHandler
  end
```